lxd/device/errors.go

```diff
@@ -18,3 +18,6 @@ var ErrUnsupportedDevType = UnsupportedError{msg: "Unsupported device type"}
 
 // ErrCannotUpdate is the error that occurs when a device cannot be updated.
 var ErrCannotUpdate = fmt.Errorf("Device does not support updates")
+
+// ErrMissingVirtiofsd is the error that occurs if virtiofsd is missing.
+var ErrMissingVirtiofsd = fmt.Errorf("Virtiofsd missing")
```

Comment:
@monstermunchkin making `ErrMissingVirtiofsd` a 'normal' error will break the functionality that was previously dependent on the type of the error being  `UnsupportedError` (on line https://github.com/lxc/lxd/pull/8826/files#diff-b488e1668ea6c7654666e0a16ffc0731cc56eb193459d9265ec9ba4ca07a8f3bL313). So the sentinel error should retain the same error type otherwise VMs won't be able to start it virtiofsd is missing.